src/test/java/net/sf/marineapi/nmea/io/SentenceReaderTest.java

```diff
@@ -141,7 +141,6 @@ public void testStartAndStop() {
 			assertNotNull(sentence);
 			assertTrue(started);
 			assertFalse(paused);
-			assertFalse(stopped);
```

Comment:
At this moment reader can be in a stopped state because it has parsed all the data.